main.tf

```diff
@@ -147,3 +147,37 @@ module "app_ingress" {
     module.app_aks,
   ]
 }
+
+module "wandb" {
+  source  = "wandb/wandb/helm"
+  version = "1.1.0"
+
+  depends_on = [ module.cert_manager ]
+
+  spec = {
+    values = {
+      global = {
+        host = local.url
+
+        storage = { connectionString = local.bucket }
+
+        mysql = {
+          name     = module.database.database_name
+          user     = module.database.username
+          password = module.database.password
+          database = module.database.database_name
+          host     = module.database.address
+          port     = 3306
+        }
+      }
+
+      ingress = {
+        issuer = { create = true }
+        class  = "azure/application-gateway"
+      }
+
+      redis = { install = true }
```

Comment:
If set to `true`, what does this do? Does this spin up a redis instance in Azure?

Reply:
Yeah, it looks like we don't have redis in azure. I don't want to make this pr too big by adding it